### DISCUSSION TITLE
Welcome to Bochs Discussions!

### DISCUSSION THREAD

**Discussion by stlintel** [bochs-emu/Bochs#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by therealdreg** [comment]
Hello! I'm Dreg, and I have a passion for everything related to OS internals, microprocessors, microcontrollers, reversing, low-level... 

I find Bochs to be fantastic. I use it regularly and have a couple of projects that utilize Bochs for some very useful tasks (at least for me):

https://github.com/therealdreg/ida_bochs_windows
https://github.com/therealdreg/bochs_linux_kernel_debugging
https://github.com/therealdreg/x86osdev
https://github.com/therealdreg/symseghelper
https://github.com/therealdreg/pdbdump_bochs

I would like to continue contributing to the Bochs ecosystem and to the Bochs debugger itself. It's a pleasure for me to learn from the Bochs team, and I hope to be of use.

**Comment by fysnet** [comment]
Hi, I am Ben Lunt.  

I have been a long time user of and contributor to this fine emulator we call Bochs.  I am proud to have been the creator of the initial implementation of the USB emulation, and am grateful for all the additions that have been made by others.

My main interest is this USB hardware. My work can be found at https://github.com/fysnet/Bochs-USB-Edition. I plan to periodically create pull requests so that the main branch can use my changes.

With my research and development of the USB and Operating System design in general, I have written a few books on the subject, mainly to document my research. They can be found at https://www.fysnet.net/osdesign_book_series.htm

I look forward to further contributions by myself as well as many others.

Thank you to all for your efforts,
Ben

**Comment by stlintel** [comment]
Hi, I am Stanislav.

Was introduced to Bochs many years ago as student doing some CPU research work. My first task was to implement Intel MMX instructions into Bochs. Since then I have re-written most of the Bochs CPU code and kept Bochs in sync with recent CPU development by both Intel and AMD. Now I have much less time for Bochs but still a lot of dreams.
Long plans like implementing more recently introduced ISA extensions, making Bochs to use multiple threads available on host to speed up emulation, enhancing Bochs debugger and instrumentation, speeding up emulation.

Although myself I almost have no time for Bochs development recently I will be glad to welcome any new developers and guide them through the code and my thoughts, including mentoring students. I hope eventually we will find someone passionate as I was back when and move the project forward together !

Thanks,
Stanislav
